pyanaconda/ui/gui/spokes/custom.py

```diff
@@ -1917,6 +1935,12 @@ def on_remove_clicked(self, button):
 
             if dialog.deleteAll:
                 for dev in (s._device for s in page.members):
+                    # Skip the device if it is shared with other installations.
+                    if self._is_device_shared(dev):
+                        log.debug("Shared device %s will be skipped.", dev.name)
+                        continue
+
+                    # Destroy the device.
```

Comment:
Please remove also this comment. It is only repeating name of the method below.
Also I'm thinking about removing the blank line above and the same for before the return inside of `_is_device_shared` method. However this is more recommendation than requirement for merge.